capa/rules.py

```diff
@@ -338,18 +338,18 @@ def build_statements(d, scope):
 
         count = d[key]
         if isinstance(count, int):
-            return Range(feature, min=count, max=count)
+            return Range(feature, count, count, d.get("description"))
```

Comment:
i lowkey prefer the kwargs here, too, as it provides a bit more context about what the arguments are.